Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.19 as builder
+FROM golang:1.21 as builder
```

Comment:
Should we pin this to a specific version?

Reply:
Yeap! Nice catch. I've update it. Originally I thought it might be the case for other repos in the devfile org but after a quick search this was the only case.